packages/mui-joy/src/AccordionDetails/AccordionDetailsProps.ts

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { OverridableStringUnion, OverrideProps } from '@mui/types';
+import { ColorPaletteProp, SxProps, VariantProp, ApplyColorInversion } from '../styles/types';
+import { SlotProps, CreateSlotsAndSlotProps } from '../utils/types';
+
+export type AccordionDetailsSlot = 'root' | 'content';
+
+export interface AccordionDetailsSlots {
+  /**
+   * The component that renders the root.
+   * @default 'div'
+   */
+  root?: React.ElementType;
+  /**
+   * The component that renders the content.
+   * @default 'div'
+   */
+  content?: React.ElementType;
+}
+
+export type AccordionDetailsSlotsAndSlotProps = CreateSlotsAndSlotProps<
+  AccordionDetailsSlots,
+  {
+    root: SlotProps<'div', {}, AccordionDetailsOwnerState>;
+    content: SlotProps<'div', {}, AccordionDetailsOwnerState>;
+  }
+>;
+
+export interface AccordionDetailsPropsVariantOverrides {}
+export interface AccordionDetailsPropsColorOverrides {}
+
+export interface AccordionDetailsTypeMap<P = {}, D extends React.ElementType = 'div'> {
+  props: P & {
```

Comment:
There's an [ongoing effort](https://github.com/mui/material-ui/pull/38171) to move away from one letter type parameters, I think we should enforce it in Joy for new components. We should also create a PR in the future for existing one letter type parameters.

Reply:
Thanks @DiegoAndai